config/task_test.go

```diff
@@ -896,62 +920,3 @@ func TestTaskConfig_validateCondition(t *testing.T) {
 		})
 	}
 }
-
-func TestTaskConfig_FinalizeValidate(t *testing.T) {
-	// Tests the full finalize then validate process, particularly how it handles the
-	// relationship between task.services and task.condition
-	//
-	// Relationships:
-	// - When catalog-service condition's regexp is nil or empty string, test
-	//    how regexp is handled by Finalize() which impacts Validate()
```

Comment:
These test-cases are no longer relevant since regexp cannot be nil. I should have caught and removed this with the PR to require regexp